tests/phpunit/integration/Core/Storage/Data_EncryptionTest.php

```diff
@@ -28,7 +28,6 @@ public function test_encrypt() {
 		// The result is base64_encoded.
 		$encrypted       = $encryption->encrypt( 'test-value' );
 		$base_64_decoded = base64_decode( $encrypted, true );
-		$this->assertNotFalse( $base_64_decoded );
```

Comment:
This assertion shouldn't have been deleted. Looking back at the commit, I think it was accidentally included with the `phpcs:ignore` on the line above. I'll restore it 👍